TopicZero/.classpath

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Do not include .classpath in pull requests. This is local enviroment specific configuration.